packages/app/src/cli/commands/app/polaris/transform.ts

```diff
@@ -0,0 +1,60 @@
+import {transform} from '../../../services/transform.js'
+import Command from '../../../utilities/app-command.js'
+import {Flags} from '@oclif/core'
+import {globalFlags} from '@shopify/cli-kit/node/cli'
+
+export default class Transform extends Command {
+  static description = 'Code transformations for updating Polaris apps.'
+  static flags = {
+    ...globalFlags,
+    include: Flags.string({
+      hidden: false,
+      description: 'Files or directory to transform',
+      env: 'SHOPIFY_FLAG_INCLUDE',
+    }),
+    package: Flags.string({
```

Comment:
I think we can leave this out for now, until we have multiple packages which we might want to specify.

Reply:
This is useful for is we want to test a snapshot or other version of a package. For example, in Polaris we create snapshot releases before merging into `main` for release. Having this option allow the user to specify a target package to get the codemod from (ex: `@shopify/polarsi-codemods@[snapshot.version]`)